map/src/main/kotlin/edu/artic/map/carousel/TourCarouselViewModel.kt

```diff
@@ -24,7 +25,8 @@ import javax.inject.Inject
 class TourCarouselViewModel @Inject constructor(private val analyticsTracker: AnalyticsTracker,
                                                 private val objectDao: ArticObjectDao,
                                                 private val audioObjectDao: ArticAudioFileDao,
-                                                val tourProgressManager: TourProgressManager) : BaseViewModel() {
+                                                val tourProgressManager: TourProgressManager,
+                                                private val languageSelector: LanguageSelector) : BaseViewModel() {
```

Comment:
Optional: put the `languageSelector` on the line above, with the other `private val`s. It'll make the git diff cleaner.